src/screens/MainScreen/index.js

```diff
@@ -20,10 +21,25 @@ const MainScreen = () => {
     getCalls();
   }, [currentUser]);
 
+  const reset = () => {
+    resetFlake(currentUser.id);
+    alert("Done! We're glad you're still interested!");
+  };
+
   return (
     <Container>
       {currentUser ? (
-        <CallsScreen callList={callList} name={currentUser.firstName} />
+        <>
+          {currentUser.flake ? (
+            <Banner
+              onClick={() => reset()}
+              text="We've noticed you missed attending some of your calls. If you still wish to participate in this session of Corona is Blind, click on this banner."
+            ></Banner>
+          ) : (
+            <></>
+          )}
```

Comment:
Instead of using the ternary operating with nothing on the other end, use {currentUser.flake && ... }

Reply:
done.